src/api/hooks/videosConnection.ts

```diff
@@ -0,0 +1,18 @@
+import { QueryHookOptions } from '@apollo/client'
+import {
+  useGetVideosConnectionQuery,
+  GetVideosConnectionQueryVariables,
+  GetVideosConnectionQuery,
+} from '@/api/queries/__generated__/videos.generated'
+
+type Opts = QueryHookOptions<GetVideosConnectionQuery>
+const useVideosConnection = (variables?: GetVideosConnectionQueryVariables, opts?: Opts) => {
```

Comment:
I think we will need to provide default sorting here to make sure we get newest videos first